README.md

```diff
@@ -64,6 +64,15 @@ known anomalies) that you would like to see incorporated into NAB.
 The NAB version will be updated whenever new data (and corresponding labels) is
 added to the corpus; NAB is currently in v1.0.
 
+#### Additional Scores
+
+Here are the NAB scores for some additional flavors of HTM. HTM detector with tm_cpp uses the implementation of temporal memory in https://github.com/numenta/nupic.core/blob/master/src/nupic/algorithms/TemporalMemory.hpp. HTM detector with no likelihood uses the raw anomaly scores directly.
```

Comment:
After this PR is merged, I'll do a PR with some code for "no likelihood" and also update this comment.